doc_template/visual_behavior_optical_physiology.rst

```diff
@@ -0,0 +1,274 @@
+behavioral contexts, including familiar and novel stimuli, as well as active 
+behavior and passive viewing conditions. This dataset can be used to evaluate 
+the influence of experience, expectation, and task engagement on neural 
+coding, interactions, and dynamics.
+
+.. image:: /_static/visual_behavior_2p/datasets.png
+   :align: center
+
+We used single- and multi-plane imaging approaches to record the activity 
+of populations of excitatory neurons and two inhibitory classes, Somatostatin 
+(Sst) and Vasoactive Intestinal Peptide (Vip) expressing interneurons, across 
+multiple cortical depths and visual areas.  
+
+[FIND AND ADD GOOD VERSION OF CRE LINE FOVS IMAGE]
+
+While 2-photon imaging data was acquired in well-trained mice, the full 
+behavioral training history of all imaged mice is also provided, allowing 
+investigation into task learning, behavioral strategy, and inter-animal 
+variability.
+
+[BEHAVIOR FIGURE OF SOME KIND]
+
+Overall, the dataset includes neural and behavioral measurements from 82 
+mice, including 3021 behavior training sessions and 551 in vivo 2-photon 
+imaging sessions, resulting in longitudinal recordings from 34,619 
+cortical cells. 
+
+The table below describes the numbers of mice, sessions, and unique recorded 
+neurons for each transgenic line and imaging platform in the dataset:
+
+.. image:: /_static/visual_behavior_2p/variants_table.png
+   :align: center
+
+GETTING STARTED
+---------------
+
+First, `install or update the AllenSDK <https://allensdk.readthedocs.io/en/latest/install.html>`_, 
+our Python based toolkit for accessing and working with Allen Institute datasets.
+
+Data is provided in in `NWB <https://www.nwb.org/>`_ format and can be downloaded using the AllenSDK, 
+or accessed directly via this S3 bucket. Regardless of which method of file 
+download you choose, we recommend that you load and interact with the data 
+using the tools provided in the AllenSDK, which have been designed to simplify 
+data access and subsequent analysis.
+
+To get started, check out these jupyter notebooks to learn how to:
+
+1) Download data using the AllenSDK or directly from our Amazon S3 bucket
+2) Identify experiments of interest using the dataset manifest
+3) Load and visualize data from a 2-photon imaging experiment
+4) Load and visualize data from a behavior training session
+5) Examine the full training history of one mouse
+6) Align neural data timeseries with stimulus and behavior events (Doug’s notebook)
+7) Get data from all imaging planes in one multi-plane imaging session and look at correlations across areas
+8) Get all data acquired for one 2-photon field of view and examine how cell activity changes between sessions
+
+For a quick summary of experimental design and data access, see the 
+`cheat sheet <https://brainmapportal-live-4cc80a57cd6e400d854-f7fdcae.divio-media.net/filer_public/0f/5d/0f5d22c9-f8f6-428c-9f7a-2983631e72b4/neuropixels_cheat_sheet_nov_2019.pdf>`_.
```

Comment:
This is from Neuropixels.

Reply:
@matchings can we get an updated link?

Reply:
@Matyasz We dont have a cheat sheet yet, but I can make one. Will work on it now.